docs/references/javascript/clerk/handle-navigation.mdx

```diff
@@ -37,7 +37,7 @@ Upon successful verification, the method will figure out if the sign in or sign
 
 Additionally, the `handleMagicLinkVerification()` method allows you to execute a callback if the successful verification happened on another device.
 
-In case the magic link verification wasn't successful, the `handleMagicLinkVerification()` method will throw a [`MagicLinkError`](https://clerk.com/docs/authentication/custom-flows/magic-links). You can check the error's `code` property to see if the magic link expired, or the verification simply failed.
+In case the magic link verification wasn't successful, the `handleMagicLinkVerification()` method will throw a [`MagicLinkError`](/docs/authentication/custom-flows/magic-links). You can check the error's `code` property to see if the magic link expired, or the verification simply failed.
```

Comment:
```suggestion
In case the magic link verification wasn't successful, the `handleMagicLinkVerification()` method will throw a [`MagicLinkError`](/docs/custom-flows/magic-links). You can check the error's `code` property to see if the magic link expired, or the verification simply failed.
```